### ISSUE TITLE
Lighting estimation for WebXR

### ISSUE THREAD

**Issue by NellWaliczek** [immersive-web/lighting-estimation#1]
There's already an API on the web for Lighting Estimation (https://w3c.github.io/ambient-light/) but it currently only supports getting the ambient intensity.
It would be useful for improving the realism of AR rendering to have slightly more information than that.  For example the color temperature would be helpful and is generally available on the underlying XR platforms that would back WebXR. There are also other more advanced lighting options available on some platforms that would be worth investigating as optional features.  One example is this from ARKit: https://developer.apple.com/documentation/arkit/arenvironmentprobeanchor

**Comment by TrevorFSmith** [comment]
@klausw Yes, once we create a feature repo all conversation should happen there. So, I recommend creating a new Issue over there and if necessary refer back to this Issue.

**Comment by kearwood** [comment]
May I suggest that SH (Spherical Harmonics) be considered as part of the format to express the environment lighting in a directional dependent manner?  

https://en.wikipedia.org/wiki/Spherical_harmonics

This has the advantage of being small (feasible to calculate and store hundreds on a mobile device), scalable (can easily access various levels of detail), and used ubiquitously in game engines for this purpose.

**Comment by kearwood** [comment]
For background on use in lighting specifically:
https://en.wikipedia.org/wiki/Spherical_harmonic_lighting

**Comment by kearwood** [comment]
Even more specifically, we could use PRT (Precomputed Radiance Transfer):

https://en.wikipedia.org/wiki/Precomputed_Radiance_Transfer

It should be possible to express a simple ambient / color temperature value in this format as well as extract such a value from PRT data that includes the higher frequency domain.
